docs/release_process.md

```diff
@@ -135,3 +136,25 @@ We're happy to announce that [AWX version 21.1.0](https://github.com/ansible/awx
 We're happy to announce that [AWX Operator version 0.22.0](https://github.com/ansible/awx-operator/releases/tag/0.22.0) is now available!
 
 ## Send the same IRC message (less the @newsbot) to #awx:ansible.com
+
+## Revert a Release
+
+Here are the steps needed to revert an AWX and an AWX-Operator release. Depending on your use case, follow the steps for reverting just an AWX release, an Operator release or both.
+
+1. Navigate to the [AWX Release Page](https://github.com/ansible/awx/releases) and delete the AWX Release that needs to be removed.
+
+![Revert-1-Image](img/revert-1.png)
+
+2. Navigate to the [AWX Operator Release Page]() and delete the AWX-Operator release that needss to tbe removed.
+
+![Revert-2-Image](img/revert-2.png)
+
+3. Navigate to [quay.io](https://quay.io/repository/ansible/awx?tag=latest&tab=tags) and delete the published AWX image(s) and tags.
+
+4. Navigate to [quay.io](https://github.com/ansible/awx-operator/releases) and delete the published AWX Operator image(s) and tags.
+
+5. Navigate to the [Ansible Galaxy Collections](https://galaxy.ansible.com/awx/awx) website and remove the published AWX collection with the bad tag.
+
+6. Navigate to the [PyPi](https://pypi.org/project/awxkit/#history) and delete the bad AWX tag and release that got published.
```

Comment:
Did we confirm that we can actually do this? If not maybe we just need to say something like "If items were released we need to just do another z stream release".

Reply:
we can certainly mention both methods (this cleanup and a fall-forward method).